builder/go.mod

```diff
@@ -0,0 +1,43 @@
+module github.com/hashicorp/syntax/builder
+
+go 1.17
+
+require (
+	github.com/ghodss/yaml v1.0.0
```

Comment:
This looks unused - do you mind running `go mod tidy -compat=1.17` to get rid of it?